lib/elixir/src/elixir.hrl

```diff
@@ -33,6 +33,8 @@
   static_atoms_encoder=nil,
   preserve_comments=nil,
   identifier_tokenizer=elixir_tokenizer,
+  ascii_identifiers_only=true,
+  unicode_warnings=nil,
```

Comment:
Why do we need the `nil` field? If it is only for recursion, then it is probably best to double check the warnings are handled on all termination clauses instead (it should be two or three clauses anyway).

Reply:
I'll do that -- was just for termination; will revert to the 2 clauses where I had that in ref. impl.